### PR TITLE
[UXP-22043] - Upgrade sp-action-bar to v0.37.0

### PR DESCRIPTION
**Pull request by inasirhussain**
Fixes - https://jira.corp.adobe.com/browse/UXP-22043

### CODE REVIEW

File: projects/swc-starter-webpack/manifest.json [hunk 1]
```diff
@@ -9,27 +9,27 @@
...
-            "height": 300
-          },
-          "preferredFloatingSize": {
-            "width": 230,
-            "height": 300
-          },
-          "label": {
-            "default": "Starter Panel"
-          }
+            "type": "panel",
```
Comment (deepthi-majji) [inline projects/swc-starter-webpack/manifest.json:12]: this format is already added in the menu PR.

File: projects/swc-starter-webpack/src/extended_swc_samples/my-action-bar.js [hunk 2]
```diff
@@ -18,7 +18,10 @@ class MyActionBar extends ActionBar {
         ActionBar.styles,
         css`
             :host([open]) {
-                background-color: antiquewhite;
+                background-color: darkgray;
+                padding: 10px;
+                border-radius: 10px;
+                box-shadow: 0px 0px 30px darkgrey;
```
Comment (deepthi-majji) [inline projects/swc-starter-webpack/src/extended_swc_samples/my-action-bar.js:24]: Why are we adding border-radius and box-shadow for extended sample?

File: packages/action-bar/src/uxp-action-bar.css [hunk 3]
```diff
@@ -11,3 +11,77 @@ governing permissions and limitations under the License.
...
+    );
+    margin-left: auto;
+}
+
+.action-group ::slotted([slot='buttons']:not(:last-child)) {
+    margin-right: var(
+        --mod-actiongroup-horizontal-spacing-regular,
+        var(--spectrum-actiongroup-horizontal-spacing-regular)
+    );
+}
```
Comment (deepthi-majji) [inline packages/action-bar/src/uxp-action-bar.css:87]: are focus related styles working for action bar?

File: projects/swc-starter-webpack/manifest.json (hunk 1 above)
Comment (inasirhussain) [reply projects/swc-starter-webpack/manifest.json:12]: let it be, which ever PR gets merged first, for the subsequent one, i will resolve it.

File: projects/swc-starter-webpack/src/extended_swc_samples/my-action-bar.js (hunk 2 above)
Comment (inasirhussain) [reply projects/swc-starter-webpack/src/extended_swc_samples/my-action-bar.js:24]: This is just to create my custom `my-action-bar` extended from `sp-action-bar` where I am trying to create an extra box-shadow around the` sp-action-bar` to make `my-action-bar` look different from the `sp-action-bar`.

File: packages/action-bar/src/uxp-action-bar.css (hunk 3 above)
Comment (inasirhussain) [reply packages/action-bar/src/uxp-action-bar.css:87]: action-bar itself doesn't has any focus styles rather contained elements like clear-button and action button receive the focus when acted upon. Currently issue with buttons is their that focus ring is not being drawn properly that exist inside it as well. That we are going to handle separately when focus issue of button is picked.

**Review by deepthi-majji — APPROVED** [review]
LGTM

**Pull request merged**